src/posix/platform/openthread-posix-config.h

```diff
@@ -435,4 +435,8 @@
 #define OPENTHREAD_POSIX_CONFIG_TMP_STORAGE_ENABLE 1
 #endif
 
+#ifndef OPENTHREAD_POSIX_CONFIG_RESOLV_CONF_ENABLED_ON_INIT
+#define OPENTHREAD_POSIX_CONFIG_RESOLV_CONF_ENABLED_ON_INIT (!OPENTHREAD_POSIX_CONFIG_ANDROID_ENABLE)
+#endif
```

Comment:
nit: how about dropping "ON", since this is the initialization value for `mIsResolvConfEnabled`?
```suggestion
#ifndef OPENTHREAD_POSIX_CONFIG_RESOLV_CONF_ENABLED_INIT
#define OPENTHREAD_POSIX_CONFIG_RESOLV_CONF_ENABLED_INIT (!OPENTHREAD_POSIX_CONFIG_ANDROID_ENABLE)
#endif
```

Reply:
Sounds good. Applied.